bundles/managed-api-service/1.17.0/manifests/managed-api-service.clusterserviceversion.yaml

```diff
@@ -0,0 +1,719 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  annotations:
+    alm-examples: |-
+      [
```

Comment:
containerImages seems to be gone :/

Reply:
Does it normally get added by the prepare-release script? Is there some configuration I'm missing that it didn't generate it?